packages/next/src/client/components/router-reducer/ppr-navigations.ts

```diff
@@ -535,14 +595,26 @@ function createCacheNodeOnNavigation(
         prefetchDataChildren !== null
           ? prefetchDataChildren[parallelRouteKey]
           : null
+      const existingSegmentMapChild =
+        existingCacheNodeChildren !== null
+          ? existingCacheNodeChildren.get(parallelRouteKey)
+          : undefined
       const segmentChild = routerStateChild[0]
       const segmentPathChild = segmentPath.concat([
         parallelRouteKey,
         segmentChild,
       ])
       const segmentKeyChild = createRouterCacheKey(segmentChild)
+
+      const existingCacheNodeChild =
+        existingSegmentMapChild !== undefined
+          ? existingSegmentMapChild.get(segmentKeyChild)
+          : undefined
```

Comment:
These could be simplified by writing:
```suggestion
      const existingCacheNodeChild = existingSegmentMapChild?.get(segmentKeyChild)
```

Reply:
This is just my style preference inherited from the React repo; I typically don’t use wider type checks than are necessary, i.e. this checks just for undefined, not null. For the same reason I tend to not use optional types and ==.